conferences/2025-agu.md

```diff
@@ -37,3 +37,42 @@ We're excited to hear what you have to say at our session!
 ## Invited speakers
 
 _TBD_
+
+
+## Organizers
+
+
+### Primary Convener: Qiusheng Wu
+
+_TODO: Insert mini-bio_
+
+
+### Convener: Max Jones
+
+_TODO: Insert mini-bio_
+
+
+### Convener: Chelle Gentemann
+
+_TODO: Insert mini-bio_
+
+
+### Convener: Tyler Erickson
+
+_TODO: Insert mini-bio_
```

Comment:
@tylere please suggest a bio! Can you please also provide a link or paste a photo into the comments here?

Reply:
Dr. Tyler Erickson is the founder of the geospatial consultancy VorGeo, a Technical Fellow at the non-profit Radiant Earth, and a founding editorial board member of the Cloud-Native Geospatial Forum. Previously he led the developer relations team for Google Earth Engine. Dr. Erickson's work focuses on building communities & ecosystems around emerging technologies, with an emphasis on technologies that can be leveraged by environmental sustainability projects. His writings & presentations can be found at https://www.analyze.earth/

![Tyler Erickson](https://media.licdn.com/dms/image/v2/D5603AQHLWPmrk6VHcQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1726186563133?e=1757548800&v=beta&t=mNpM6EfWf76P6_O2HoeQfz9t0KWGeqO_ALHLnV6YqAw)

Reply:
Thank you!